### PR TITLE
Add plotting evoked potentials for individual channels

### PR DESCRIPTION
**Pull request by hofaflo**
As discussed in #256, this adds a menu entry "Plot evoked...", leading to the following dialog:
image

If the montage is not set, the topomaps-box is greyed out:
image

### CODE REVIEW

**Comment by cbrnr** [comment]
I was wondering if we should use a simple checkbox for the GFP. The "Only" option looks kind of weird, and probably no one is going to need such a plot. I guess we can always change it back later if someone really needs it.

**Comment by hofaflo** [comment]
Good idea, that makes many things simpler!
New dialog:
image

File: mnelab/viz.py [hunk 1]
```diff
@@ -48,6 +48,68 @@ def plot_erds(data, freqs, n_cycles, baseline, times=(None, None)):
...
+        The time point(s) to plot. If `"auto"`, 5 evenly spaced topographies
+        between the first and last time instant will be shown. If `"peaks"`,
+        finds time points automatically by checking for 3 local maxima in
+        Global Field Power.
+
+    Returns
+    -------
+    list[matplotlib.figure.Figure]
+        A list of the figure(s) generated.
+    """
```
[the remark this answers is not in the document]
Comment (hofaflo) [reply mnelab/viz.py:89]: Sure, I'd add pydocstyle as a pre-commit check and reformat all docstrings in a new PR?

[the remark this answers is not in the document]
Comment (cbrnr) [reply mnelab/viz.py:89]: Perfect!

**Comment by cbrnr** [comment]
Thanks @hofaflo!

**Pull request merged**